src/appshell/view/appmenumodel.cpp

```diff
@@ -132,7 +132,7 @@ MenuItem* AppMenuModel::makeFileMenu()
     MenuItemList fileItems {
         makeMenuItem("file-new"),
         makeMenuItem("file-open"),
-        makeMenu(TranslatableString("appshell", "Open &recent"), recentScoresList, "menu-file-open", openRecentEnabled),
+        makeMenu(TranslatableString("appshell/file", "Open &recent"), recentScoresList, "menu-file-open", openRecentEnabled),
```

Comment:
I quite like this, since it is also helpful for translators to find the string. I thought, maybe we can make it even clearer by making the context `"appshell/menubar/file"` etc. 
(I think "menubar" is the clearest, better than something like "appmenu" or "mainmenu", what do you think?)

Reply:
I'm not sure how helpfil the `appshell` part is. But just `menu` should be enough, this string here is not in the menubar itself too, its 'parent', "&File" is, this is ie below that.